src/NeustaPimcorePresentationBundle.php

```diff
@@ -3,31 +3,92 @@
 namespace Neusta\Pimcore\PresentationBundle;
 
 use Pimcore\Extension\Bundle\AbstractPimcoreBundle;
+use Pimcore\Extension\Bundle\Installer\InstallerInterface;
 use Pimcore\Extension\Bundle\Traits\PackageVersionTrait;
+use Pimcore\Version;
 
-class NeustaPimcorePresentationBundle extends AbstractPimcoreBundle
-{
-    use PackageVersionTrait;
-
-    public function getPath(): string
+if (!method_exists(Version::class, 'getMajorVersion') || 10 === Version::getMajorVersion()) {
```

Comment:
If this is just because of the assets, it would probably be easier to register them through the listener instead.

Reply:
It is mainly for the assets, but also for the return type on `getInstaller`. I will look at the listener solution